Bio/PDB/Dice.py

```diff
@@ -42,8 +42,8 @@ def accept_residue(self, residue):
             # skip HETATMS
             return 0
         if icode != " ":
-            warnings.warn("WARNING: Icode at %s" % residue.get_id(),
-                          RuntimeWarning)
+            warnings.warn("WARNING: Icode %s at position %s" \
```

Comment:
You don't need the trailing slash here, the brackets are enough to tell Python it is multi-line.